xskillscore/core/resampling.py

```diff
@@ -172,7 +174,13 @@ def resample_iterations_idx(
       for interannual-to-decadal predictions experiments. Climate Dynamics, 40(1–2),
       245–272. https://doi.org/10/f4jjvf
     """
-    # equivalent to above
+    if dim not in forecast.coords:
+        # raise ValueError(f"dim={dim} needs to be coordinate.")
```

Comment:
can this be deleted?

Reply:
Yep. That’s an alternative but I prefer that it just works